deploy/notary.go

```diff
@@ -93,22 +93,13 @@ func enableNotary(ctx context.Context, prm enableNotaryPrm) error {
 
 		prm.logger.Info("checking Notary role of the committee members...")
 
-		accsWithNotaryRole, err := roleContract.GetDesignatedByRole(noderoles.P2PNotary, prm.monitor.currentHeight())
+		ok, err := checkRole(noderoles.P2PNotary, roleContract, prm.monitor, prm.committee)
 		if err != nil {
-			prm.logger.Error("failed to check role of the committee, will try again later", zap.Error(err))
+			prm.logger.Error("failed to check notary role of the committee, will try again later", zap.Error(err))
```

Comment:
```suggestion
			prm.logger.Error("failed to check Notary role of the committee, will try again later", zap.Error(err))
```
to follow other messages